### PR TITLE
Add cli option `--quite`

### PR DESCRIPTION
**Pull request by alexander-akait**
@bevacqua 

I can rewrite all code on promise and update deps if agree, i can PR or i can maintenance this repo (if you give me access) :smile:

### CODE REVIEW

File: bin/spritesmith [hunk 1]
```diff
@@ -12,6 +12,7 @@ var multi = require('multi-glob');
 var spritesmith = require('..');
 var formats = require('../lib/formats');
 var argv = require('minimist')(process.argv.slice(2));
+var quite = argv.quite || false;
```
Comment (sankhasp) [inline bin/spritesmith:15]: its spelled quiet

Comment (alexander-akait) [inline bin/spritesmith:15]: @sankhasp true default?

Comment (alexander-akait) [inline bin/spritesmith:15]: /cc @sankhasp

**Pull request closed without merging**